grafana-plugin/src/containers/IntegrationContainers/ExpandedIntegrationRouteDisplay/ExpandedIntegrationRouteDisplay.tsx

```diff
@@ -72,7 +83,12 @@ const ExpandedIntegrationRouteDisplay: React.FC<ExpandedIntegrationRouteDisplayP
     );
 
     useEffect(() => {
-      escalationChainStore.updateItems();
+      (async function () {
```

Comment:
do you really need async here?

Reply:
Right, it can go away. Previously I was awaiting for a promise